lib/ngsiHandlers.js

```diff
@@ -176,11 +204,12 @@ function ngsiExecuteHandler(id, type, attributes, callback) {
                 device.type,
                 device.apikey,
                 attribute.name,
-                'pruebas',
+                // Ezequiel: mandar el valor del ack como resultado de la órden.
```

Comment:
Please, use only english in comments that will be kept in the code. I don't think using names in comments is appropriate, also.